recipes/ipydatetime/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ex
+
+$PYTHON -m pip install . -vv
+npm pack jupyter-widget-datetime@${PKG_VERSION}
```

Comment:
the `sdist` from pypi includes this tarball already, and will already be deployed to the correct location from with `pip install`. I recommend dropping this altogether, as then this can be a proper `noarch: python` package.